src/main/scala/ru/org/codingteam/horta/configuration/Configuration.scala

```diff
@@ -41,6 +44,10 @@ object Configuration {
   lazy val login = properties.getProperty("login")
   lazy val password = properties.getProperty("password")
   lazy val server = properties.getProperty("server")
+  lazy val xmppTimeout = {
+    val ms = Option(properties.getProperty("xmpp.timeout_ms")).getOrElse("5000")
+    Integer.parseInt(ms).milliseconds
```

Comment:
Why not something like?

```scala
Option(properties.getProperty("xmpp.timeout_ms"))
  .map(Integer.parseInt(_))
  .getOrElse(5000)
  .milliseconds
```

Doesn't invoke `Integer.parseInt()` if there is no `xmpp.timeout_ms`

Furthermore `Integer.parseInt` can fail with `NumberFormatException`. Are we gonna ignore that fact for now?